package.json

```diff
@@ -56,5 +57,8 @@
             "name": "DHIS2 Application Runtime",
             "description": "A singular runtime dependency for applications on the [DHIS2 platform](https://platform.dhis2.nu)"
         }
+    },
+    "resolutions": {
+        "@dhis2/app-runtime": "2.10.0-pwa.4"
```

Comment:
I'm not sure if this should still be there. I suspect it shouldn't.

Reply:
Nice find - this has now been fixed by https://github.com/dhis2/app-runtime/pull/984. Could you update this branch with `git merge master`?

Reply:
Thanks! Have merged in the new commits.